packages/ses/scripts/hermesTransforms.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { parse } from '@babel/parser';
+import babelGenerate from '@agoric/babel-generator';
+import babelTraverse from '@babel/traverse';
+import * as t from '@babel/types';
+
+// @ts-expect-error TS2339 Property 'default' does not exist
+const traverse = babelTraverse.default || babelTraverse;
+const generate = babelGenerate.default || babelGenerate;
+
+const decoder = new TextDecoder('utf-8');
+// @ts-expect-error TS2554 Expected 0 arguments, but got 1
+const encoder = new TextEncoder('utf-8');
```

Comment:
afaik `TextEncoder` does not accept an argument and always uses `utf-8` ([ref](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder))